src/dcqc/file.py

```diff
@@ -105,7 +105,7 @@ def get_file_type(cls, file_type: str) -> FileType:
 FileType("TXT", (".txt",), "format_1964")
 FileType("JSON", (".json",), "format_3464")
 FileType("JSON-LD", (".jsonld",), "format_3749")
-FileType("TIFF", (".tif", ".tiff"), "format_3591")
+FileType("TIFF", (".tif", ".tiff", ".svs"), "format_3591")
```

Comment:
Can you also test and add `scn`? cc @adamjtaylor

Reply:
added and tested successfully. I updated the description to reflect the change.